code/modules/cm_marines/vehicle_part_fabricator.dm

```diff
@@ -29,6 +32,7 @@
 /obj/structure/machinery/part_fabricator/dropship/ui_data(mob/user)
 	return list(
 		"points" = get_point_store()
```

Comment:
missing ,

did you test this?

Reply:
I did some untested changes on it recently, trying to finde out how UI works. it is draft for a reason